src/js/vendor/wasm_exec.js

```
@@ -73,6 +73,14 @@
 		}
 	}
 
+	if (!globalThis.path) {
```

ESLint

[error] Unexpected tab character.

(no-tabs)